src/UsersModule/Components/Navbar/Navbar.tsx

```diff
@@ -1,7 +1,151 @@
-import React from 'react'
+import React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import CssBaseline from '@mui/material/CssBaseline';
+import Divider from '@mui/material/Divider';
+import Drawer from '@mui/material/Drawer';
+import IconButton from '@mui/material/IconButton';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemText from '@mui/material/ListItemText';
+import MenuIcon from '@mui/icons-material/Menu';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import {   useNavigate } from "react-router-dom";
+import Link from '@mui/material/Link';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { toast } from "react-toastify";
+
 
 export default function Navbar() {
```

Comment:
We have two NavBar components in our application,  we need to have a unique name for each one to be able to easily distinguish between them.